File: backend/Entities/Models/Romaneio.cs
using System;
using System.Collections.Generic;

namespace Entities.Models;

public partial class Romaneio : BaseEntity
{
    public Guid Id { get; set; }

    public Guid TenantId { get; set; }

    public DateTime DataEmissao { get; set; }

    public DateTime DataCadastro { get; set; }

    public virtual RomaneioCarga? RomaneioCarga { get; set; }

    public virtual ICollection<RomaneioDocumento> RomaneioDocumentos { get; } = new List<RomaneioDocumento>();

    public virtual Tenant Tenant { get; set; } = null!;
}

File: frontend-with-menus/src/app/Business/Models/romaneio.ts
import { RomaneioCarga } from "./romaneioCarga";
import { RomaneioDocumento } from "./romaneioDocumento";
import { Tenant } from "./tenant";


export interface Romaneio  {
    id: string;
    tenantId: string;
    dataEmissao: string;
    dataCadastro: string;
    romaneioCarga: RomaneioCarga | null;
    romaneioDocumentos: RomaneioDocumento[];
    tenant: Tenant;
}